website/docs/cloud-docs/vcs/index.mdx

```diff
@@ -116,4 +116,4 @@ Viewing VCS events requires permission to manage VCS settings for the organizati
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-![Screenshot: the VCS Event Viewer UI, showing sample events and filtering capability](/img/docs/vcs-event-viewer.png)
+To view VCS events for your organization, go to your organization's **Settings* and click **Events** under **Version Control**. The **VCS Events** page appears.
```

Comment:
Okay so I think if we're not telling folks to click something and we're kind of talking generally about the settings, we should do it this way, without bold. I believe that this aligns with Rita's Learn PR, but please push back if I'm wrong there. Also, I've seen them (without warning) reorganize the settings in the sidebar. So I'm hesitant to say what section it's in, in case they rearrange stuff or change the title in the future. Just less to update. BUT let me know if you think it's too unclear this way - if so, I support you putting it back.
```suggestion
To view VCS events for your organization, go to your organization's settings and click **Events**. The **VCS Events** page appears.
```